README.md

```diff
@@ -74,9 +74,9 @@ matrix:
 
 `yafti` is the uBlue "first boot" installer. It shows up the first time a user logs into uBlue. By default, the menu also shows up again anytime the image's yafti configuration differs from the user's last encounter, so feel free to expand or modify your custom image's yafti configuration over time. Your users will then see the yafti menu again after the OS update, and will be given a chance to install any new additions.
 
-Its configuration can be found in `/etc/yafti.yml` of the installed OS. It includes an optional selection of Flatpaks to install, along with a new group that's automatically added for all Flatpaks declared in `recipe.yml`. You can look at what's done in the config and modify it to your liking.
+Its configuration can be found in `/usr/etc/yafti.yml` of the installed OS. It includes an optional selection of Flatpaks to install, along with a new group that's automatically added for all Flatpaks declared in `recipe.yml`. You can look at what's done in the config and modify it to your liking.
```

Comment:
Keeping these as `/etc/` was actually intentional, as on a functional system these files _should_ exist in `/etc/` in order for everything to function. But eh, both work here.

Reply:
Yeah it was a misunderstanding from me. I didn't realize until today that Silverblue-based operating systems clone and merge `/usr/etc -> /etc` on deployment. Really sorry about that. But glad to hear that the notes still make sense.

Reply:
@EinoHR Oh and yeah, it just hit me: Either way, it's good to tell users to look at the `/usr` path of the installed OS. They should **NOT** be editing the `/etc` versions of the files, because that could break OS updates, since the files would be marked as "manually edited" and might not get properly auto-updated anymore.